mbedtls/src/x509/certificate.rs

```diff
@@ -225,12 +227,12 @@ impl Certificate {
         err_info: Option<&mut String>,
         cb: Option<F>,
         expected_common_name: Option<&str>,
-    ) -> Result<()>
+    ) -> std::result::Result<(), (Error, VerifyError)>
```

Comment:
Hi @s-arash, since `rust-mbedtls` need to support no_std.
So we need to use `core::result::Result` instead of `std::result::Result`

Reply:
I see! Will fix.